tests/Http/CampaignTest.php

```diff
@@ -199,6 +199,54 @@ public function testUpdatingACampaign()
         ]);
     }
 
+    /**
+     * Test for updating a campaign successfully with contentful campaign id.
+     *
+     * PATCH /api/v3/campaigns/:campaign_id
+     * @return void
+     */
+    public function testUpdatingACampaignWithContentfulId()
+    {
+        // Create a campaign to update.
+        $campaign = factory(Campaign::class)->create();
+
+        // Update the contentful campaign id.
+        $response = $this->withAdminAccessToken()->patchJson('api/v3/campaigns/' . $campaign->id, [
+            'contentful_campaign_id' => '123456',
+        ]);
+
+        // Make sure the campaign update is persisted.
+        $response = $this->getJson('api/v3/campaigns/' . $campaign->id);
+
+        $response->assertStatus(200);
+
+        $response->assertJson([
+            'data' => [
+                'contentful_campaign_id' => '123456',
+            ],
+        ]);
```

Comment:
I think we want to make sure this got actually saved to the database too. Something like this:
```
        $this->assertDatabaseHas('campaigns', [
            'id' => $campaign->id,
            'contentful_campaign_id' => 123456,
        ]);
```